app/schema.graphql

```diff
@@ -202,6 +208,9 @@ type EmbeddingDimension implements Node {
 
     """HDBSCAN cluster selection epsilon"""
     clusterSelectionEpsilon: Float! = 0
+
+    """Data quality metric to be computed on each cluster"""
+    dataQualityMetric: DataQualityMetricInput
```

Comment:
we can move this lower to the `Cluster` no?

Reply:
yea, we could do that, that way it'll be sent back together with the response

Reply:
the only reason i kept it out of the response is to keep the response more or less generic in case we need to expand beyond the data quality metrics in the future

Reply:
Oh I think I meant to say that we probably need to expose this as a parameter so that this is possible:

```graphql
{
   cluster {
       driftRatio
       metric: dataQualityMetricComparison(metric: DQMetricInput!) {
           primaryValue
           referenceValue
       }
   }
}

Reply:
ok, just to confirm: this is to create a standalone query endpoint (called `cluster`) with a list of event id's as the input?

Reply:
Like if you define `dataQualityMetric` on `Cluster` (the node) you satisfy both cases in one go.

Reply:
So it would look like
```
type Cluster {
   driftRatio: float
   dataQualtityMetric(metric: ClusterDataQualityMetricInput): DatasetValues
}
type Query {
  model: Model!
  node(id: GlobalID!): Node!
  hdbscanClustering(
    """Event ID of the coordinates"""
    eventIds: [ID!]!
  ): [Cluster!]!
}